notary/src/proxy.rs

```diff
@@ -119,19 +121,16 @@ pub fn create_tee_proof(
   response: &NotaryResponse,
   State(state): State<Arc<SharedState>>,
 ) -> Result<TeeProof, NotaryServerError> {
-  let validation_result = validate_notarization_legal(manifest, request, response)?;
-
-  let manifest_hash = manifest.to_keccak_digest()?;
-  let extraction_hash = validation_result.extraction_keccak_digest()?;
-  let proof_value_hash = keccak_digest(&[manifest_hash, extraction_hash].concat());
-
-  let to_sign = VerifyOutput {
-    value:    format!("0x{}", hex::encode(proof_value_hash)),
-    manifest: manifest.clone(),
-  };
-  let signature = sign_verification(to_sign, State(state))?;
-  let data = TeeProofData { manifest_hash: manifest_hash.to_vec() };
-
+  validate_notarization_legal(manifest, request, response)?;
+  let path = manifest.response.body.json_path().clone();
+  let body = serde_json::from_slice(&response.notary_response_body.clone().body.unwrap()).unwrap();
```

Comment:
This is still breaking for the HTML format. Shall we disable HTML support for the time, or reconsider with this approach: https://github.com/pluto/web-prover/pull/563#discussion_r1992977823 ?